lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_strings_file_validation_helper.rb

```diff
@@ -73,12 +72,10 @@ def self.find_duplicated_keys(file:)
 
           File.readlines(file).each_with_index do |line, line_no|
             line.chars.each_with_index do |c, col_no|
-              # Handle escaped characters at a global level. This is more
-              # straightforward than having a `TRANSITIONS` table that account
-              # for it.
+              # Handle escaped characters at a global level.
+              # This is more straightforward than having a `TRANSITIONS` table that account for it.
```

Comment:
Completely unrelated to the "change hard-wrap to soft-wrap" thing but just realized that this comment (of mine) was not super clear (i.e. this might suggest that "we don't have a `TRANSITIONS` table, and would have to add one to account for that" as opposed to the intended meaning of "we would have to account for it in the _existing_ `TRANSITIONS` table); so could be the occasion to improve it to:
```suggestion
              # This is more straightforward than having to account for it in the `TRANSITIONS` table.
```